apps/omg/test/omg/dependency_conformance/signature_test.exs

```diff
@@ -43,24 +40,41 @@ defmodule OMG.DependencyConformance.SignatureTest do
     {:ok, [addr | _]} = Ethereumex.HttpClient.eth_accounts()
 
     {:ok, _, signtest_addr} =
-      Eth.Deployer.create_new(OMG.Eth.Eip712SignatureWrapper, root_path, Eth.Encoding.from_hex(addr), [])
+      Eth.Deployer.create_new(OMG.Eth.PaymentEip712LibMock, root_path, Eth.Encoding.from_hex(addr), [])
```

Comment:
first time seeing this... so is this deploying one special contract and you're identifying that contract by providing `OMG.Eth.PaymentEip712LibMock` module? why a module?

Reply:
by inertia and convention I guess, but the convention broke anyway now.

If this is confusing, I'll poke around how to straighten out